<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input a tweet:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTweet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Tweet</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtAccountOne">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>104</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtAccountTwo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>104</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
